src/Configuration/ConfigurationFactory.php

```diff
@@ -151,10 +159,33 @@ public function create(
             self::retrieveMinCoveredMsi($minCoveredMsi, $schema),
             $msiPrecision,
             $threadCount,
-            $dryRun
+            $dryRun,
+            $ignoreSourceCodeMutatorsMap
         );
     }
 
+    /**
+     * @param array<string, mixed> $schemaMutators
+     *
+     * @return array<class-string<Mutator&ConfigurableMutator>, mixed[]>
+     */
+    public function resolveMutators(array $schemaMutators, string $mutatorsInput): array
```

Comment:
```suggestion
    private function resolveMutators(array $schemaMutators, string $mutatorsInput): array
```
I don't see it used anywhere else.

Reply:
thanks, fixed